crankshaft.sh

```diff
@@ -4,12 +4,10 @@
 # Written by Huan Truong <htruong@tnhh.net>, 2018
 # This script is licensed under GNU Public License v3
 
-IMAGE_FILE=2018-03-13-raspbian-stretch-lite.zip
-IMAGE_FILE_UNZIPPED=2018-03-13-raspbian-stretch-lite.img
+IMAGE_FILE=raspbian-stretch-lite.zip
 TODAY_EXT=$(date +"%Y-%m-%d")
 IMAGE_FILE_CUSTOMIZED=${IMAGE:-"crankshaft-${TODAY_EXT}.img"}
-IMAGE_URL=http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2018-03-13/2018-03-13-raspbian-stretch-lite.zip
-IMAGE_SIZE_RAW=1858076672
+IMAGE_URL=https://downloads.raspberrypi.org/raspbian_lite_latest
 IMAGE_ROOTPART_START=98304
```

Comment:
If we want to make it more universal, we need to re-import the trick from raspbian to figure out where rootfs starts. Otherwise with other versions of raspbian lite, this will be wrong.

Reply:
on it's way :-) Will add some more small tweaks now and re-push the commit if ready